doc/common/roadmap.rst

```diff
@@ -0,0 +1,124 @@
+Roadmap
+=======
+
+Overview
+--------
+
+-  The activities of the Anuket Project community are articulated around `Projects <#project>`__ (sub-projects of Anuket Project sometimes also referred to as Workstreams (WS)), `Milestones <#milestones>`__ and `Releases <#release>`__.
+-  The Anuket Project is embracing simultaneous delivery model, meaning that all contributing projects have to follow the cadence and intermediate milestones.
+-  Each Anuket Project release is the only delivery vehicle and is common for all projects.
+-  The Anuket Project current release plan is available `here <https://wiki.anuket.io/display/HOME/Releases>`__.
+
+.. image:: ./figures/Release-Lifecycle.png
+   :alt: "Figure 1: Milestones"
+
+**Figure 1:** Milestones
+
+**Definitions**
+
+A project (aka WS) is:
```

Comment:
This and some of the other above referenced (see below) are not section headings and so wouldn't have implicit references - in the markdown version names were created:

`Projects <#project>`__  `Milestones <#milestones>`__ and `Releases <#release>`__

Reply:
Are there any links pointing to here?

I'm a bit lost now with the implicit header anchoring. In my view they work only until we use links to .md files and use the relative link corrector extension. Later we can use [sphinx.ext.autosectionlabel](https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html) or we can just add labels manually. Any opinions or preferences? @pgoyal01 , @karinesevilla , @gkunz , @collivier ?